plugins/services/src/js/structs/Framework.js

```diff
@@ -18,6 +23,23 @@ module.exports = class Framework extends Application {
     this._spec = null;
   }
 
+  /**
+   * @override
+   */
+  getImages() {
```

Comment:
I fear that this is too expensive to be called every 2 seconds, we should memoize it 🤔

Reply:
+1 for memorization

Reply:
You are right @Fabs the reason we decided to go this path is that is more performant, meaning not calling every 2 secs.